js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/addGraph/useEdgeCollectionOptions.tsx

```diff
@@ -0,0 +1,49 @@
+import { CollectionType } from "arangojs";
+import useSWR from "swr";
+import { getCurrentDB } from "../../../utils/arangoClient";
+
+const useFetchCollections = () => {
+  const currentDB = getCurrentDB();
+  const fetchCollections = async () => {
+    const collections = await currentDB.listCollections();
+    const sortedCollection = collections.sort((a, b) => {
```

Comment:
What do you think about using a localCompare?

```
const sortedCollection = collections.sort((a, b) => a.name.localeCompare(b.name, undefined, { sensitivity: 'base' }));
```

Reply:
makes sense, will handle UTF-8 stuff better!